src/Avalonia.Controls/TopLevel.cs

```diff
@@ -185,7 +185,7 @@ IStyleHost IStyleHost.StylingParent
         }
 
         /// <inheritdoc/>
-        IRenderTarget IRenderRoot.CreateRenderTarget()
+        public virtual IRenderTarget CreateRenderTarget()
```

Comment:
It was my initial thought, but complier was against that. Can't implement interface if member isn't public, blablabla